pom.xml

```diff
@@ -1658,7 +1659,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-shade-plugin</artifactId>
-                    <version>2.2</version>
+                    <version>3.2.4</version>
```

Comment:
Can you just override the version in the extension POM instead of globally?

Reply:
I asked @IvanVan to make the change global rather than dealing with multiple shade plugin versions